src/ec/eddsa.rs

```diff
@@ -234,6 +234,24 @@ impl Point {
     }
 }
 
+// Keep this in sync with `ge_p2` in curve25519/internal.h.
+#[repr(C)]
+struct ProjPoint {
```

Comment:
In `ring::ec::suite_b::ops` projective points are named just `Point`. I think it's better to name this `Point` too, renaming the four-component one.